dask_geomodeling/geometry/sources.py

```diff
@@ -153,3 +158,100 @@ def process(url, request):
                     )
 
             return {"projection": request["projection"], "features": f}
+
+
+class GeometryWKTSource(GeometryBlock):
+    """Converts a single geometry to a geometry source
+
+    Args:
+      wkt (string): the WKT representation of a geometry
+      projection (string): the projection of the geometry
+
+    Returns:
+      GeometryBlock
+    """
+
+    def __init__(self, wkt, projection):
+        if not isinstance(wkt, str):
+            raise TypeError("'{}' object is not allowed".format(type(wkt)))
+        if not isinstance(projection, str):
+            raise TypeError("'{}' object is not allowed".format(type(projection)))
+        try:
+            load_wkt(wkt)
+        except WKTReadingError:
+            raise ValueError("The provided geometry is not a valid WKT")
+        try:
+            utils.get_sr(projection)
+        except TypeError:
+            raise ValueError("The provided projection is not a valid WKT")
+        super().__init__(wkt, projection)
+
+    @property
+    def wkt(self):
+        return self.args[0]
+
+    @property
+    def projection(self):
+        return self.args[1]
+
+    @property
+    def columns(self):
+        return {"geometry"}
+
+    def get_sources_and_requests(self, **request):
+        data = {"wkt": self.wkt, "projection": self.projection}
+        return [(data, None), (request, None)]
+
+    @staticmethod
+    def process(data, request):
+        if request.get("filters"):
+            raise ValueError("Filter are not supported")
```

Comment:
Maybe best to just ignore the "filters" argument. There is no column anyway to filter on.